### DISCUSSION TITLE
SVG Example not loading the SVG(tiger) image in other system.

### DISCUSSION THREAD

**Discussion by mayankjoshii** [hecrj/iced#1816] [Q&A]
I tried building the svg example present in the iced libaray. The system in which it was build, it is working as exepected (i.e., the tiger SVG image is loading) but if I'm trying to run it in different machine(laptop), then the SVG image is not loading and only white screen is created. 

Please help, how should I proceed in this case: 

Working case: Machine 1( where this app was built)
![image](https://user-images.githubusercontent.com/98211866/235288566-f10a3601-dc0d-4c39-8466-58a3569884fd.png)

Failure case: Machine 2(where I  was testing this app)

![image](https://user-images.githubusercontent.com/98211866/235288650-c6145fc4-3190-4190-a9d8-df2f6446e423.png)

**Comment by jessbowers** [comment]
The issue is that the tiger image is being loaded from a hardcoded path in the binary.

```
 let handle = svg::Handle::from_path(format!(
            "{}/resources/tiger.svg",
            env!("CARGO_MANIFEST_DIR")
        ));
```

so you'll need to put a tiger image on the machine you're running it on, in that same directory path. Or, alternatively change the code to load from a path specified in an env var.
